### PR TITLE
Add introductory material recommended from sub0

### PR DESCRIPTION
**Pull request by jamesray1**

### CODE REVIEW

**Comment by bkchr** [comment]
We can list `substrate.dev` directly in the Readme, but not a link for every tutorial. 

@shawntabrizi any idea where we can put the `substrate.dev` in the readme?

File: README.adoc [hunk 1]
```diff
@@ -168,6 +168,8 @@ You can distribute `mychain.json` so that everyone can synchronize and (dependin
 
 == Building
 
+See https://tiny.cc/substrate-getting-started.
```
Comment (gavofyork) [inline README.adoc]: i don't really want to like to a google-hosted slides doc here.

**Comment by jamesray1** [comment]
Ok I just thought it would be helpful to specifically refer to this tutorial because it was frequently directed to at the Sub0 workshop.

**Comment by jamesray1** [comment]
Alternatively just backup the slides locally as a PDF, etc., and host it on IPFS, so that Google can't censor the material (which it has been known to do with Ethereum YouTube meetings).

**Comment by jamesray1** [comment]
Also, another reason that I added the link to the getting started slides rather substrate.dev is because it contains the links to the other slides on the final slide, and that's the presentation to get started with substrate!

**Comment by jamesray1** [comment]
Also, another reason that I added the link to the getting started slides rather than substrate.dev is because it contains the links to the other slides on the final slide, and that's the presentation to get started with substrate!

**Comment by jamesray1** [comment]
On second thought, a link to https://docs.substrate.dev/ should be enough.

**Pull request closed without merging**